app.rb

```diff
@@ -0,0 +1,171 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'person'
+
+class App
+  def initialize
+    @books = []
+    @users = []
+    @rentals = []
+  end
+
+  def books_list
+    list = ''
+    @books.each_with_index { |book, i| list << "\n[#{i + 1}]. Title: #{book.title}, Author: #{book.author}" }
+    list << "\n"
+  end
+
+  def users_list
+    list = ''
+    @users.each_with_index do |user, i|
+      list << "\n[#{i + 1}]. id:#{user.id} class:#{user.class}  name: #{user.name}, age: #{user.age}"
+    end
+    list << "\n\n"
+  end
+
+  def rentals_list
+    list = ''
+    @rentals.each_with_index do |rental, i|
+      list << "\n[#{i + 1}]. date:#{rental.date} book:#{rental.book.title} owner: #{rental.person.name}"
+    end
+    list << "\n\n"
+  end
```

Comment:
- [ ] **Good job so far 👏 :clap:** However, It would be better to make a small adjustment on listing the rentals, currently I am inputting the user id and the app shows nothing, 😉😉 Kindly fix this and make it easier for the user to see the rentals as given in the[ video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) 🙏 
https://www.loom.com/share/9dc72d1d98d94194ac3eebb5050b4ef2